packages/components/src/box-control/all-input-control.tsx

```diff
@@ -74,6 +74,7 @@ export default function AllInputControl( {
 		<>
 			<StyledUnitControl
 				{ ...props }
+				__shouldNotWarnDeprecated36pxSize
```

Comment:
There also seem to be affected UnitControl usages in BorderControl and FontSizePicker.

Reply:
Hi @mirka, Just a question, How to identify this instances on the other components, Because it is not necessary I am seeing that it always use `UnitControl`, it may be else as show in the comment, `StyledUnitControl`.So can you please provide a way (if It's there, other than checking individual components), although I can be sure it could be an experience magic as well 😅.

Reply:
Great question 😆 For these specifically, searching the codebase for `__next40pxDefaultSize={ __next40pxDefaultSize }` might be the fastest way to spot affected instances. It includes all components but the list is short enough to scan manually.

It's ok if we miss a few spots, that's what the code review is for.

Reply:
Thanks for that 😄.